gtsfm/data_association/point3d_initializer.py

```diff
@@ -55,6 +56,39 @@ class TriangulationParam(Enum):
     RANSAC_TOPK_BASELINES = 3  # deterministically choose hypotheses with largest estimate baseline
 
 
+class TriangulationOptions(NamedTuple):
+    """Options for triangulation RANSAC solver."""
+
+    reproj_error_threshold: float
+    min_track_len: int
+    mode: TriangulationParam
+
+    # RANSAC parameters
+    min_inlier_ratio: float = 0.5
```

Comment:
I'm more accustomed to "inlier_ratio" rather than "min_inlier_ratio" when computing the number of RANSAC hypotheses -- what does `min_inlier_ratio` mean here?

Reply:
Yes, the name is actually misleading (I was following COLMAP's implementation). It's actually the assumed outlier ratio, i.e., a higher value will result in more iterations in order to achieve the defined confidence.

I will change the name and add a description to the docstring.

Reply:
Thanks